scorecard/profile_data/indicators/cash_coverage.py

```diff
@@ -39,6 +40,27 @@ class CashCoverage(SeriesIndicator):
             ")",
         ],
     }
+    formula_v2 = {
+        "text": "= Cash available at year end / Operating Expenditure per month",
+        "actual": [
+            "=", 
+            {
+                "cube": "cflow_v2",
+                "item_codes": ["0430"],
+                "amount_type": "AUDA",
+            },
+            "/",
+            "(",
+            {
+                "cube": "incexp_v2",
+                "item_codes": ["2900"],
+                "amount_type": "ADJB",
```

Comment:
where did item 2900 ADJB come from?

As far as I can tell, this formula is actually using the sum of item 2000-3000 AUDA:

![Screenshot_2022-04-12_14-07-50](https://user-images.githubusercontent.com/235801/162959856-75dd9019-7715-4f33-b754-161a4c3beee5.png)
![Screenshot_2022-04-12_14-08-06](https://user-images.githubusercontent.com/235801/162959872-884078a2-318b-4694-9180-183fc899fdbc.png)

meaning all these items should be mentioned and highlighted

![Screenshot_2022-04-12_14-10-14](https://user-images.githubusercontent.com/235801/162959903-3e54498c-dc8c-472d-b4bf-c06a9b6b85d2.png)

Reply:
2800 should probably also be included - I think that is _operating_ transfers while 3300 and 3400 are capital transfers

Reply:
Added the item codes including 2800 and changed ADJB to AUDA